docs/website/content/docs/bootstrapping.md

```diff
@@ -84,72 +137,82 @@ EOF
-  namespace: test
-spec:
-  template:
-    spec:
-      serverClassRef:
-        apiVersion: metal.sidero.dev/v1alpha1
-        kind: ServerClass
----
-apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
-kind: TalosControlPlane
+## Create Server Class
+
+We must now create a server class to wrap our servers we registered.
+This is necessary for using the Talos control plane provider for Cluster API.
+The qualifiers needed for your server class will differ based on the data provided by your registration flow.
+See the [server class docs](/docs/serverclasses/) for more info on how these work.
+
+Here is an example of how to apply the server class once you have the proper info:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: metal.sidero.dev/v1alpha1
+kind: ServerClass
 metadata:
-  name: management-plane-cp
-  namespace: test
+  name: default
 spec:
-  version: v1.18.1
-  replicas: 1
-  infrastructureTemplate:
-    kind: MetalMachineTemplate
-    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-    name: management-plane-cp
-    namespace: test
-  controlPlaneConfig:
-    init:
-      generateType: init
-    controlplane:
-      generateType: controlplane
+  qualifiers:
+    cpu:
+      - manufacturer: Intel(R) Corporation
+        version: Intel(R) Atom(TM) CPU C3558 @ 2.20GHz
+EOF
+```
+
+Once created, you should see the servers that make up your server class appear as "available":
+
+```bash
+$ kubectl get serverclass
+NAME      AVAILABLE                                  IN USE
+default   ["00000000-0000-0000-0000-d05099d33360"]   []
+```
+
+## Create Management Plane
+
+We are now ready to template out our management plane.
+Using clusterctl, we can create a cluster manifest with:
+
+```bash
+clusterctl config cluster management-plane -i sidero > management-plane.yaml
 ```
+
+Note that there are several variables that should be set in order for the templating to work properly:
+
+- CONTROL_PLANE_ENDPOINT
+- CONTROL_PLANE_SERVERCLASS
+- WORKER_SERVERCLASS
+- KUBERNETES_VERSION
+
+Now that we have the manifest, we can simply apply it:
+
+```bash
+kubectl apply -f management-plane.yaml
+```
+
+Once the management plane is setup, you can fetch the talosconfig with a command like:
+
+```bash
+kubectl get talosconfig -o yaml management-plane-cp-xxx -o jsonpath='{.status.talosConfig}' > management-plane-talosconfig.yaml
+```
+
+Then the management plane's kubeconfig can be fetched with `talosctl --talosconfig management-plane-talosconfig.yaml kubeconfig /desired/path`
+
+## Pivoting
+
+Once we have the kubeconfig for the management cluster, we now have the ability to pivot the cluster from our bootstrap.
+Using clusterctl, issue:
+
+```bash
+  clusterctl move --to-kubeconfig=/path/to/management-plane/kubeconfig
+```
+
+Upon completion of this command, we can now tear down our bootstrap cluster with `talosctl cluster destroy` and begin using our management plane as our point of creation for all future clusters!
```

Comment:
should we patch Sidero components in the management cluster for new API endpoint and stuff like that?

do we need to re-setup DHCP/TFTP, as endpoints got changed?

Reply:
Yep, those will be coming in a "post-pivot"/"creating your first cluster" guide that follows this one.